src/build_infrastructure/android/README.md

```diff
@@ -0,0 +1,69 @@
+## Usage
+
+Start from the CDK root, where this `README.md` lives:
+```console
+cd ~/amplify-ci-support/src/build_infrastructure_resources/android
```

Comment:
Is this still the right path? Or did `_resources` get stripped after this written?

Reply:
Ahhh...I changed the name after I updated the readme.